test/gtest/ucp/ucp_test.h

```diff
@@ -74,7 +79,7 @@ class ucp_test_base : public ucs::test_base {
     protected:
         ucs::handle<ucp_context_h> m_ucph;
         std::vector<ucs::handle<ucp_worker_h> >  m_workers;
-        std::vector<ucs::handle<ucp_ep_h> >      m_eps;
+        ep_map_t m_eps;
```

Comment:
the reason is to keep them in same data structure instead of lookup from one array to another based on pointer
the vector also contains only initialized instances (it grows on demand), and the top-level vector of workers has to exist anyway